### ISSUE TITLE
Handle repeat SBI

### ISSUE THREAD

**Issue by CrispinF** [exeGesIS-SDM/ELM#8]
Handle repeat applications for the same SBI in some manner, depending on RPA requirements. Scenarios could include:

1. One SBI appearing more than once in the list entered in one user request.
2. An SBI entered again in a different request, at any time after the SBI was previously requested.

This needs careful thought about real world scenarios. and requirements.

**Comment by sarahpoppy** [comment]
Yes, but handling should be as simple as possible - there could be a valid reason why a SBI has been requested a second time.

**Comment by CrispinF** [comment]
Allow users to request a given SBI more than once (no checks).
But when they come to collect, they should be presented only with the latest one.

**Comment by epennifold** [comment]
Being dealt with as part of #19.
